### ISSUE TITLE
useSeoMeta: 'articleTag' just take in account the first item of the array

### ISSUE THREAD

**Issue by lousolverson** [nuxt/nuxt#26048]
### Environment

- Operating System: `Darwin`
- Node Version:     `v21.5.0`
- Nuxt Version:     `3.10.1`
- CLI Version:      `3.10.0`
- Nitro Version:    `2.8.1`
- Package Manager:  `yarn@1.22.21`
- Builder:          `-`
- User Config:      `devtools`, `runtimeConfig`, `ssr`, `hooks`, `nitro`, `ignore`
- Runtime Modules:  `-`
- Build Modules:    `-`

### Reproduction

https://stackblitz.com/edit/github-pd4p7k?file=app.vue

### Describe the bug

I use the useSeoMeta composable. When I want to populate the 'articleTag' property with an array (e.g. '['foo','bar']), this results in the creation of the same tag, as many times as the length of the array, but always with the same value (the first). In this case it will give this:
<meta property="article:tag" content="foo">
<meta property="article:tag" content="foo">

### Additional context

```
<head>
...
<meta property="article:tag" content="foo">
<meta property="article:tag" content="foo">
...
</head>
```

**Comment by danielroe** [comment]
cc @harlan-zw

**Comment by manniL** [comment]
The values are both existent (see screenshot), so the main issue is the generation of the first item multiple times.
![image](https://github.com/nuxt/nuxt/assets/640208/34687539-25f8-4cb1-be78-0b79b20356e5)

**Comment by lousolverson** [comment]
> The values are both existent (see screenshot). Items are generated multiple times though: ![image](https://private-user-images.githubusercontent.com/640208/309539250-34687539-25f8-4cb1-be78-0b79b20356e5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk0NjA1NzIsIm5iZiI6MTcwOTQ2MDI3MiwicGF0aCI6Ii82NDAyMDgvMzA5NTM5MjUwLTM0Njg3NTM5LTI1ZjgtNGNiMS1iZTc4LTBiNzliMjAzNTZlNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMwM1QxMDA0MzJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MzdkYjRjNzBiNzU2YmI0MDVkYTFiOWI1M2ZmOWQ3ODUxZTEwMzgyOGQ4YmJiMGE1Y2RiYjI4OGY4MWM3OWNkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.c-hMGH_UaPwHScGyIjRVShsY4zwLBrPXoNWjfG1Ao7E)
> 
> With three items `['foo', 'test', 'bar']`
> 
> ![image](https://private-user-images.githubusercontent.com/640208/309539351-802a0e32-2365-4938-83cd-8b79bbdf12a7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk0NjA1NzIsIm5iZiI6MTcwOTQ2MDI3MiwicGF0aCI6Ii82NDAyMDgvMzA5NTM5MzUxLTgwMmEwZTMyLTIzNjUtNDkzOC04M2NkLThiNzliYmRmMTJhNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMwM1QxMDA0MzJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMzIyNjQzNDBiMmZiNGNiZGQwZmM0MzM0MzlkNjIzMGEyNDgzYWNlOTRjODgxZmUxOTk5YjA2YmY0NDVmNjgwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.78-F45N-VdYfqRfv7HYVDimihKKEoWYe7jEW8Mq6ig4)

you're right, I didn't notice it

**Comment by danielroe** [comment]
My guess is that the meta property/content should be enough to provide a key/deduplicate the meta tags, and this is an issue to raise upstream in `unhead`: https://github.com/unjs/unhead

However, you can probably work around by using `useHead` with an explicit key.

**Comment by harlan-zw** [comment]
Thanks for the issue, this should be fixed in v1.8.11.
